lib/mountain_view/presenter.rb

```diff
@@ -0,0 +1,70 @@
+module MountainView
+  class Presenter
+    class_attribute :_attributes, instance_accessor: false
+    self._attributes = {}
+
+    attr_reader :slug, :properties
+    alias_method :props, :properties
+
+    def initialize(slug, properties={})
+      @slug = slug
+      @properties = attribute_defaults.deep_merge(properties)
+    end
+
+    def partial
+      "#{slug}/#{slug}"
+    end
```

Comment:
What do you think of naming this method `to_partial_path`? Ref: https://robots.thoughtbot.com/rendering-collections-in-rails 

It's a bit more Rails-y, and that way we can use `render component, locals: component.locals` in the `render_component` method.